### ISSUE TITLE
Interested in owning ours?

### ISSUE THREAD

**Issue by jrconlin** [googleapis/google-cloud-rust#14]
Hi,

A while ago, we needed a library to talk to spanner and a few other GCP services. We worked with some folk over at GCP to get [a rust library](https://github.com/mozilla-services/google-cloud-rust/) that builds off of the GRPC models made. We've been the custodian for it since, mostly because, well, nobody else wanted it, and we had the need.

I'll admit, it feels a bit weird for us to support this, and while it works and does what we need, I wouldn't call it well polished. Hopefully, it's enough to get your effort properly kick-started at least.

(Sorry for not reaching out earlier, but I just discovered this effort yesterday.)

**Comment by coryan** [comment]
Sorry for the late response. We only recently restated work on a Google Cloud SDK for Rust.

The short answer is no. We have  chosen a different approach from the one taken by mozilla-services/google-cloud-rust.  We did look at that implementation (and many others!) before starting to build a SDK from scratch.  The decision to build a new SDK is a reflection on the features or quality of the existing SDKs that the community has developed.

We are sure those other SDKs are great.  We hope to make ours so good that people will want to migrate.

As to the rationale.  I cannot share a lot of details, but in brief: we have needs for consistent tooling across SDKs in different languages, we have plans beyond Rust, and we have requirements around software supply chain that are easier to meet if the bulk of the code is developed and reviewed by Google employees.
